src/main/java/org/openbase/bco/bcozy/view/InfoPane.java

```diff
@@ -77,62 +91,107 @@ public Label getMouseOverText() {
+        return show(identifier, "-fx-text-fill: red;");
+    }
+
+    public static InfoPaneConfigurer show(final String identifier, String style) {
+        Objects.requireNonNull(identifier);
+        Objects.requireNonNull(style);
+
         try {
-            final ObserverLabel textLabel = getInstance().textLabel;
+            final InfoPane infoPane = getInstance();
+            infoPane.resetTimeline();
+
             Platform.runLater(() -> {
-                if (BCozy.baseColorIsWhite) {
-                    textLabel.setStyle("-fx-text-fill: white; -fx-font-size: 16;");
-                } else {
-                    textLabel.setStyle("-fx-text-fill: black; -fx-font-size: 16;");
-                }
-                textLabel.setIdentifier(identifier);
+                infoPane.setBackground(null);
+                infoPane.textLabel.setStyle(style + "-fx-font-size: 16;");
+                infoPane.textLabel.setIdentifier(identifier);
             });
+            return new InfoPaneConfigurer();
         } catch (CouldNotPerformException ex) {
             ExceptionPrinter.printHistory("Could not print user feedback!", ex, LOGGER);
         }
+        return null;
     }
 
-    public static void confirmation(final String identifier) {
-        assert identifier != null;
+    /**
+     * Hides the InfoPane.
+     */
+    public static void hide() {
         try {
-            final ObserverLabel textLabel = getInstance().textLabel;
-            Platform.runLater(() -> {
-                textLabel.setStyle("-fx-text-fill: green; -fx-font-size: 16;");
-                textLabel.setIdentifier(identifier);
-            });
+            show("", "");
+            getInstance().setBackground(null);
+            getInstance().resetTimeline();
         } catch (CouldNotPerformException ex) {
             ExceptionPrinter.printHistory("Could not print user feedback!", ex, LOGGER);
         }
     }
 
-    public static void warn(final String identifier) {
-        assert identifier != null;
-        try {
+    private void resetTimeline() throws NotAvailableException {
+        if (this.timeline != null) {
+            this.timeline.stop();
+        }
+        this.timeline = null;
+    }
 
-            final ObserverLabel textLabel = getInstance().textLabel;
-            Platform.runLater(() -> {
-                textLabel.setStyle("-fx-text-fill: orange; -fx-font-size: 16;");
-                textLabel.setIdentifier(identifier);
-            });
+    /**
+     * Used to hide InfoPane after certain time with Builder-like style.
+     */
+    public static class InfoPaneConfigurer {
 
-        } catch (CouldNotPerformException ex) {
-            ExceptionPrinter.printHistory("Could not print user feedback!", ex, LOGGER);
+        //Prevent instantiation from other classes
+        private InfoPaneConfigurer() {
         }
-    }
 
-    public static void error(final String identifier) {
```

Comment:
API brake: why did you remove this method?

Reply:
same here with the return type.